docs/readmes/howtos/inbound_roaming.md

```diff
@@ -210,3 +236,51 @@ For better details Federated Gateway logs:
 - Restart docker process, so the vars are taken `sudo docker-compose down` and
   `sudo docker-compose up -d`
 - Display the logs using for example`sudo docker-compose logs -f s8_proxy`
+
+### Test with s6a_cli and s8_cli
+Feg has a couple of clients to run an HSS Authentication Request (s6a) and
+Create Session Request (s8) without the need of having a UE. You can run them
+either on FEG or AGW.
+
+- Run From FEG
+```
+# use feg s6a_proxy
+sudo docker-compose exec s8_proxy /var/opt/magma/bin/s6a_cli air -remote_s6a 001002000000810
+# use s6a_porxy that runs on the cli
+sudo docker-compose exec s8_proxy /var/opt/magma/bin/s6a_cli air -use_builtincli false  -remote_s6a 001002000000810
+
+# use feg s8_proxy
+sudo docker-compose exec s8_proxy /var/opt/magma/bin/s8_cli cs -server 192.168.32.118:2123 -delete 3
+# use s8_porxy that runs on the cli
+sudo docker-compose exec s8_proxy /var/opt/magma/bin/s8_cli cs -server 192.168.32.118:2123 -use_builtincli false -delete 3
+```
+
+- Run from AGW
+```
+# extract the binaries from docker container from feg, and move them to AGW
+sudo docker cp s6a_proxy:/var/opt/magma/bin/s6a_cli .
+sudo docker cp s8_proxy:/var/opt/magma/bin/s8_cli .
+```
+
+```
+# execute from AGW
+.s6a_cli air -remote_s6a 001002000000810
+.s8_cli cs -server 192.168.32.118:2123 -delete 3 -apn inet -remote_s8 001002000000810
+```
+
+## Configuration Example
```

Comment:
Throughout: only `#` is Title Cased, everything else (`##` and below) are Sentence cased

https://magma.github.io/magma/docs/next/contributing/contribute_conventions#markdown

Reply:
Corrected, but left the names of the networks in capital